website/CONTRIBUTE.md

```diff
@@ -0,0 +1,290 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+
+# Contribution Guide
+
+This guide consists of:
+
+- [Project structure](#project-structure)
+- [How to add a new blogpost](#how-to-add-a-new-blogpost)
+- [How to add a new page](#how-to-add-a-new-page)
+- [How to write in Hugo way](#how-to-write-in-hugo-way)
+  - [Define TableOfContents](#define-tableofcontents)
+  - [Language switching](#language-switching)
+  - [Code highlighting](#code-highlighting)
+  - [Adding class to markdown text](#paragraph)
+  - [Table](#table)
+  - [Github sample](#github-sample)
+  - [Others](#others)
+- [Translation guide](#translation-guide)
+
+## Project structure
+
+```
+www/
+├── dist                                  # bundle files
+├── site
+│   ├── archetypes                        # frontmatter template
+│   ├── assets
+│   │   └── scss                          # styles
+│   ├── content                           # pages
+│   │   └── en
+│   │       ├── blog
+│   │       ├── community
+│   │       ├── contribute
+│   │       ├── documentation
+│   │       ├── get-started
+│   │       ├── privacy_policy
+│   │       ├── roadmap
+│   │       └── security
+│   │       └── _index.md
+│   ├── data
+│   ├── layouts                           # content template
+│   ├── static
+│   │   ├── downloads                     # downloaded files
+│   │   └── fonts
+│   │   └── images
+│   │   └── js
+│   └── themes
+│       └── docsy
+├── build_github_samples.sh
+├── check-links.sh                        # links checker
+└── package.json
+```
+
+## How to add a new blogpost
```

Comment:
I think the most important thing is adding a new docs page. You should write explicitly how to do that